biz.aQute.bnd.embedded-repo/bnd.bnd

```diff
@@ -11,11 +11,15 @@ Timestamp: ${_@tstamp}
 	biz.aQute.launcher=@${repo;biz.aQute.launcher;snapshot}!/biz.aQute.launcher.pre.jar, \
 	biz.aQute.remote.launcher/biz.aQute.remote.launcher-${base.version}.jar=${repo;biz.aQute.remote.launcher;snapshot}, \
 	biz.aQute.junit/biz.aQute.junit-${base.version}.jar=${repo;biz.aQute.junit;snapshot}, \
+	biz.aQute.junit.platform/biz.aQute.junit.platform-${base.version}.jar=${repo;biz.aQute.junit.platform;snapshot}, \
+	biz.aQute.junit.platform.reporting/biz.aQute.junit.platform.reporting-${junit-platform-reporting}.jar=${repo;biz.aQute.junit.platform.reporting;snapshot}, \
```

Comment:
This must be `${base-version}` like the other Bnd jars.

Reply:
See above comments about `biz.aQute.junit.platform.reporting` and its version.